examples/certificate-check-volume.yaml

```diff
@@ -0,0 +1,48 @@
+# An example Job to run a certificate check of OpenShift's internal
+# certificate status from within OpenShift.
+#
+# It uses the openshift/openshift-ansible container image.
+# (see README_CONTAINER_IMAGE.md in the top level dir for more details).
+#
+# It expects:
+#   - the Ansible inventory file from the 'hosts' key of a ConfigMap
+#     named 'inventory'
+#   - the ssh key in the 'ssh-privatekey' key of a Secret named 'sshkey'
+#   - a PersistentVolumeClaim named 'certcheck-reports' where the
+#     generated reports are stored
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: certificate-check
+spec:
+  containers:
+  - name: openshift-ansible
+    image: openshift/openshift-ansible
+    env:
+    - name: PLAYBOOK_FILE
+      value: playbooks/certificate_expiry/html_and_json_timestamp.yaml
```

Comment:
Is this the only different between the two files? If it is, maybe it should be a single file and passing this value as an argument? Well well, when we have a single object in the YAML we cannot use parameters, but if we use a template object, then we could. Not sure what would be best for usability/maintenance. What do you think?